scripts/lib/api/generateToc.ts

```diff
@@ -100,7 +102,7 @@ function addItemsToModules(
     );
 
     if (itemModuleTitle) {
-      const itemModule = tocModulesByTitle[itemModuleTitle];
+      const itemModule = tocModulesByTitle.get(itemModuleTitle) as TocEntry;
```

Comment:
We can be confident `.get()` works because `itemModuleTitle` uses `tocModuleTitles`, which is all the `.keys()` of `tocModulesByTitle`.